selfservice/strategy/link/strategy_recovery.go

```diff
@@ -191,7 +191,8 @@ func (s *Strategy) createRecoveryLink(w http.ResponseWriter, r *http.Request, _
 			url.Values{
 				"token": {token.Token},
 				"flow":  {req.ID.String()},
-			}).String()})
+			}).String()},
+		herodot.UnescapedHTML)
```

Comment:
Could you please add a test for this?

Reply:
As I mentioned in the PR description, I wasn't sure about the best way to do it, do you have any suggestions?

Since `"\u0026" == "&"` in Go, I'm not sure how we could test it from within `checkLink`, defined here https://github.com/ory/kratos/blob/master/selfservice/strategy/link/strategy_recovery_test.go#L56.

I guess that the way to do it would involve looking at the raw body of the response, before it is decoded into `*admin.CreateRecoveryLinkOK`, but I'm not sure how to go about doing that in the current tests.

Reply:
Hm, it also appears that current tests fail due to this? Basically `\u0026` is `&` in UTF-8 speak so all clients that are capable of decoding UTF8 should know what to do here. What library where you facing issues with here?

Reply:
The original issue was observed using `curl` on the command line: the link returned was not clickable.

Reply:
I see, could you try `curl ... | jq`? I guess curl is returning the raw payload

Reply:
Yeah, that was what I did to get around it in my script :smile:

Reply:
Ok, unless we find a good solution on how to fix the tests or figure out why they fail in the first place we might have to stick with the UTF8 encoded variant, although I agree that it is a bit counterintuitive